packages/app/src/views/Home.vue

```diff
@@ -16,15 +27,121 @@
 <script>
 import CourseSearchBar from "../components/CourseSearchBar";
 import Timetable from "../components/Timetable";
+import TimetableCourseCard from "../components/TimetableCourseCard";
 import COURSES_SEARCH_BAR_QUERY from "../graphql/CoursesSearchBar.gql";
 
 export default {
   components: {
     CourseSearchBar,
-    Timetable
+    Timetable,
+    TimetableCourseCard
   },
   data() {
     return {
+      courseCards: {
```

Comment:
Let's rename this to courses. The data model should be abstracted away from the component it's rendered in.